dashboard/app/views/home/audio_test.html.haml

```diff
@@ -0,0 +1,63 @@
+!!! 5
+%html{dir: locale_dir}
+  %head
+    = javascript_include_tag 'soundEffects'
+    :javascript
+      var enableDiagnostics = true;
+      var sounds = new Sounds(enableDiagnostics);
```

Comment:
On the one hand, might come in handy with new browsers / at least for IE Mobile test tomorrow.

But on the other hand, quick enough to re-add and it'll be in source control. & will make it easier to read through